src/app/file-explorer.js

```diff
@@ -253,8 +254,6 @@ function fileExplorer (appAPI, files) {
     el.className = css.fileexplorer
     element.parentElement.replaceChild(el, element)
     element = el
-    fileFocus(filepath)
-    appAPI.switchToFile(filepath)
```

Comment:
It was retained from the original code. I'm not keen on having it. It makes sense when a new file is created with the "New file" button, however not so much when it is imported.